### PR TITLE
VS-931 exome integration test

### PR DESCRIPTION
**Pull request by gbggrant**
This PR adds an integration test for Exomes.
It also adds an optional input to the test to allow you to use the default dockers (and NOT build the gatk override jar) if you so desire.

Passing Integration test [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/e16edc16-92a7-4a52-834a-1b45e1a2f92c).

### CODE REVIEW

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 1]
```diff
@@ -74,22 +66,62 @@ workflow GvsQuickstartIntegration {
...
+        call QuickstartVcfIntegration.GvsQuickstartVcfIntegration as QuickstartVcfExomeIntegration {
+            input:
+                branch_name = branch_name,
+                is_wgs = false,
+                use_VQSR_lite = true,
+                extract_do_not_filter_override = false,
+                dataset_suffix = "exome",
+                use_default_dockers = use_default_dockers,
+                gatk_override = if (use_default_dockers) then none else BuildGATKJar.jar,
+                interval_list = full_exome_interval_list,
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:102]: Note that we aren't subsetting the Exome interval list down to 20, X, and Y. Not sure if I should do so. Not sure why I didn't. Discuss.

File: scripts/variantstore/wdl/GvsJointVariantCalling.wdl [hunk 2]
```diff
@@ -10,6 +10,9 @@ workflow GvsJointVariantCalling {
         Array[File] input_vcfs
         Array[File] input_vcf_indexes
         String call_set_identifier
+        File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
+        Boolean use_interval_weights = true
+        File interval_weights_bed = "gs://broad-public-datasets/gvs/weights/gvs_vet_weights_1kb.bed"
         String? extract_output_gcs_dir
         String drop_state = "FORTY"
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsJointVariantCalling.wdl:17]: Do we know that the exomes have been reblocked with the latest version? Or should this be 60?

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 3]
```diff
@@ -4,67 +4,59 @@ import "GvsQuickstartVcfIntegration.wdl" as QuickstartVcfIntegration
...
+        Boolean use_default_dockers = false
         Boolean run_vcf_integration = true
         Boolean run_hail_integration = true
+        Boolean run_exome_integration = true
     }
 
-    File full_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
-    File expected_output_prefix = "gs://gvs-internal-quickstart/integration/2023-07-19-quicker/"
+    File full_wgs_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
+    File full_exome_interval_list = "gs://gcp-public-data--broad-references/hg38/v0/bge_exome_calling_regions.v1.1.interval_list"
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:17]: total nit, but it's weird that we call one a wgs interval list and the other one an exome interval list

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl (hunk 1 above)
Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:102]: how many intervals are even on chrX and chrY in an exome haha

but like how much time does this even save us?

File: scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl [hunk 4]
```diff
@@ -7,37 +7,62 @@ workflow GvsQuickstartVcfIntegration {
...
+                                            "gs://gvs-internal-quickstart/reblocked-v2-vcfs/HG00418.haplotypeCalls.er.raw.vcf.gz.rb.g.vcf.gz.tbi",
+                                            ]
+        Array[File] exome_input_vcf_indexes = [
+                                               "gs://gvs-internal-scratch/ggrant/exomes_for_integration_test/NA12878.rb.g.vcf.gz.tbi",
+                                               "gs://gvs-internal-scratch/ggrant/exomes_for_integration_test/NA12891.rb.g.vcf.gz.tbi",
+                                               "gs://gvs-internal-scratch/ggrant/exomes_for_integration_test/NA12892.rb.g.vcf.gz.tbi",
+                                               ]
 
         Int? extract_scatter_count
         String drop_state = "FORTY"
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl:49]: same as before

File: scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl [hunk 5]
```diff
@@ -7,37 +7,62 @@ workflow GvsQuickstartVcfIntegration {
...
+        Array[File] wgs_input_vcfs = [
+                                     "gs://gvs-internal-quickstart/reblocked-v2-vcfs/HG00405.haplotypeCalls.er.raw.vcf.gz.rb.g.vcf.gz",
+                                     "gs://gvs-internal-quickstart/reblocked-v2-vcfs/HG00408.haplotypeCalls.er.raw.vcf.gz.rb.g.vcf.gz",
+                                     "gs://gvs-internal-quickstart/reblocked-v2-vcfs/HG00418.haplotypeCalls.er.raw.vcf.gz.rb.g.vcf.gz",
+                                     ]
+        Array[File] exome_input_vcfs = [
+                                     "gs://gvs-internal-scratch/ggrant/exomes_for_integration_test/NA12878.rb.g.vcf.gz",
+                                     "gs://gvs-internal-scratch/ggrant/exomes_for_integration_test/NA12891.rb.g.vcf.gz",
+                                     "gs://gvs-internal-scratch/ggrant/exomes_for_integration_test/NA12892.rb.g.vcf.gz",
+                                     ]
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl:35]: umm how is this going to work with bulk ingest?

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl (hunk 3 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:17]: Not sure why that's a problem/nit?

**Pull request closed without merging**